pkg/scheduler/testing/wrappers.go

```diff
@@ -18,6 +18,8 @@ package testing
 
 import (
 	"fmt"
+	policy "k8s.io/api/policy/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
```

Comment:
nit: re-sort the imports (moving them down w/ their similar imports)

Reply:
fixed.